json-server/error-rerouter.js

```diff
@@ -0,0 +1,16 @@
+const getFailRoute = (req) => {
+    return req.headers?.cookie?.match(/fail-route=(?<failRoute>\w+);/)?.groups.failRoute
```

Comment:
Is there a reason that groups didn't have optional chaining?

Reply:
If match returns a value, that value will always have a groups property